### ISSUE TITLE
building issue with 1.10.1

### ISSUE THREAD

**Issue by sennevb** [atheioschain/go-atheios#2]
make gath
build/env.sh go run build/ci.go install ./cmd/gath
ci.go:161: You have Go version go1.10.1
ci.go:162: go-atheios requires at least Go version 1.4 and cannot
ci.go:163: be compiled with an earlier version. Please upgrade your Go installation.
exit status 1
Makefile:15: recipe for target 'gath' failed
make: *** [gath] Error 1

**Comment by DigDeepMining** [comment]
Yeah duplicated, once Berran is back I'm sure he'll take a look but it might be a error our end, I'm taking a look now
build/env.sh go run build/ci.go install ./cmd/gath
ci.go:161: You have Go version go1.10.3
ci.go:162: go-atheios requires at least Go version 1.4 and cannot
ci.go:163: be compiled with an earlier version. Please upgrade your Go installation.
exit status 1
Makefile:15: recipe for target 'gath' failed
make: *** [gath] Error 1

**Comment by DigDeepMining** [comment]
Ok I fixed the issue my end...

I downgraded go to version 1.9 as the readme states 1.9> I thought I'd try that...

Worked first attempt

**Comment by atheios-berran** [comment]
Odd thing is, I built my nodes with go1.10.1 (you can see them running at http://stats.atheios.com/) I'm currently remote, but I think I see what the issue may be, going to try and do a quick fix before i have to get one the road.

**Comment by atheios-berran** [comment]
I made some updates and was able to build with go 1.10.3 on ubuntu and mac.

**Ubuntu 16.04:**
```
$ make gath
build/env.sh go run build/ci.go install ./cmd/gath
>>> /usr/local/go/bin/go install -ldflags -X main.gitCommit=3c5031047ec66a73458876e84645d81daccd7b84 -v ./cmd/gath
Done building.
Run "build/bin/gath" to launch gath.

$ go version
go version go1.10.3 linux/amd64
```


**Mac:**
```
$ make gath
build/env.sh go run build/ci.go install ./cmd/gath
>>> /usr/local/Cellar/go/1.10.3/libexec/bin/go install -ldflags -X main.gitCommit=3c5031047ec66a73458876e84645d81daccd7b84 -v ./cmd/gath
Done building.
Run "build/bin/gath" to launch gath.

$ go version
go version go1.10.3 darwin/amd64
```

**Comment by sennevb** [comment]
confirmed! working now
